src/java/htsjdk/samtools/SamStreams.java

```diff
@@ -39,6 +39,19 @@ public static boolean isCRAMFile(final InputStream stream) throws IOException {
 
         return Arrays.equals(buffer, CramHeader.MAGIC);
     }
+
+    public static boolean isSRAFile(final InputStream stream) throws IOException {
+        final byte[] MAGIC = "NCBI.sra".getBytes();
```

Comment:
Pull out the magic bytes for SRA files into a public constant in `SRAFileReader` (or wherever it would be most appropriate)